### PR TITLE
issue #30186 - disable triggers during schema mods

### PR DESCRIPTION
**Pull request by xtpclark**
proposed improvements to some scripts to speed up process on large databases

### CODE REVIEW

File: foundation-database/public/tables/shiphead.sql [hunk 1]
```diff
@@ -1,2 +1,7 @@
-﻿-- Add column
+ALTER TABLE shiphead DISABLE TRIGGER ALL;
+
+--Add column
```
Comment (bendiy) [inline foundation-database/public/tables/shiphead.sql]: I fixed the byte order marker (https://github.com/xtuple/xtuple/pull/3047/commits/dafeda1f7a5c7d5ecab165978147d57d6b2bd583) in my PR:
https://github.com/xtuple/xtuple/pull/3049

**Comment by bendiy** [comment]
I think @gilmoskowitz is addressing the triggers in another PR.

**Comment by gilmoskowitz** [comment]
Indeed I am working on this. See issue 30186.

File: foundation-database/public/tables/cntct.sql [hunk 2]
```diff
@@ -1,2 +1,6 @@
+ALTER TABLE cntct DISABLE TRIGGER ALL;
+
 select xt.add_column('cntct', 'cntct_created',      'TIMESTAMP WITH TIME ZONE', NULL, 'public');
 select xt.add_column('cntct', 'cntct_lastupdated',  'TIMESTAMP WITH TIME ZONE', NULL, 'public');
+
+ALTER TABLE cntct DISABLE TRIGGER ALL;
```
Comment (gilmoskowitz) [inline foundation-database/public/tables/cntct.sql]: This should be `enable`

Comment (xtpclark) [inline foundation-database/public/tables/cntct.sql]: fixed in da286dc

**Comment by gilmoskowitz** [comment]
Obsoleted by xtuple/xtuple#3051

**Pull request closed without merging**